src/main/java/com/openlattice/data/DataApi.java

```diff
@@ -82,13 +66,13 @@
      *            selectedProperties are not specified, all readable properties will be fetched.
      * @return An iterable containing the entity data, using property type FQNs as keys
      */
-    @POST( BASE + "/" + ENTITY_SET + "/" + SET_ID_PATH )
-    Iterable<SetMultimap<FullQualifiedName, Object>> loadSelectedEntitySetData(
+    @POST( BASE + ENTITY_SET + SET_ID_PATH )
+    Iterable<Map<FullQualifiedName, Set<Object>>> loadSelectedEntitySetData(
             @Path( ENTITY_SET_ID ) UUID entitySetId,
             @Body EntitySetSelection req,
             @Query( FILE_TYPE ) FileType fileType );
 
-    @POST( BASE + "/" + ENTITY_SET + "/" )
+    @POST( BASE + ENTITY_SET + "/" )
```

Comment:
not for this PR, but I feel like we should remove this slash at the end at some point -- requests like `/datastore/data/set/?setId=blah` feel bad

Reply:
(but will require corresponding frontend changes)

Reply:
I created a more general endpoint, where the entity set id is not a query parameter and there is no extra slash to conform with REST standards.
`List<UUID> createEntities( @Body Map<UUID, List<Map<UUID, Set<Object>>>> entities )`